examples/canvas_more/canvas_more/app.py

```diff
@@ -0,0 +1,216 @@
+from contextlib import contextmanager
+from math import pi, cos, sin, atan
+
+import toga
+from toga.style import Pack
+from toga.style.pack import COLUMN
+from travertino.size import at_least
+
+
+class ExampleCanvasApp(toga.App):
+    def startup(self):
+        # Initial width and height for the Canvas
+        iwidth = 600
+        iheight = 400
+
+        # Set up main window
+        self.main_window = toga.MainWindow(title=self.name)
+
+        # Setup the style selection box
+        self._style_select = toga.Selection(
+            items=list(self.styles), style=Pack(height=32), on_select=self.set_style
+        )
+        # Setup the canvas
+        self._canvas = toga.Canvas(style=Pack(flex=1), on_resize=self.resize_contents)
+        self._canvas.intrinsic = \
```

Comment:
It makes much more sense to me to set minimal sizes on the Canvas directly then on the surrounding window, especially when we have more widgets in it. I do intend to add that support to Pack (As this hack indicates - I've been looking at the relevant code...), so I'd rather keep it like this for now.

If you don't mind, I'll open an issue about this and add a comment here referring to it.

Reply:
ok - so, I *do* mind :-) This is a public example, and putting something in an example is an indication that it's good practice. Modifying "intrinsic" definitely isn't good practice, and I don't want to encourage it.

In this particular case (from my testing, anyway), dropping the minimum size constraint doesn't actually make the demo *worse*. Setting a minimum size definitely *improves* the demo, as it prevents reducing the window to a tiny size - but I'd consider that a nice to have, rather than an essential feature.